ui/src/app/dashboard/controllers/DashCtrl.js

```diff
@@ -2,17 +2,15 @@
 
   angular
     .module('app')
-    .controller('DashCtrl', [ '$scope', 'ToastService', 'DashService',
-      DashCtrl
-    ]);
+    .controller('DashCtrl', [ '$scope', 'ToastService', 'DashService', 'UserService', DashCtrl]);
 
-  function DashCtrl($scope, ToastService, DashService) {
+  function DashCtrl($scope, ToastService, DashService, UserService) {
     $scope.dlink = {link: ''};
     $scope.downloads = [];
-    $scope.username = localStorage.getItem('Username');
+    $scope.username = UserService.getUsername();
+    var socket;
```

Comment:
Also get the host from the configuration

https://github.com/scorelab/Bassa/blob/master/ui/src/app/index.js#L6